### DISCUSSION TITLE
Blue Hacktribe match factory samples with factory patterns

### DISCUSSION THREAD

**Discussion by uasianbolt** [bangcorrupt/hacktribe#343] [Q&A]
Hello,

I just switched my blue Electribe2 to a Hacktribe. I made a lot of patterns with the use of factory samples. These patterns kept their sequences but without the factory samples they aren't listenable. 

I already found a zip file with all the factory samples of the Electribe 2 synth chronologically arranged.

Is it possible to replace the Hacktribe samples with these factory samples? although i've already read that you can only import samples, >501.

or any other solutions out there? I would love to hear my old patterns again with factory samples.

**Comment by bangcorrupt** [comment]
https://github.com/bangcorrupt/hacktribe/wiki/FAQ#the-factory-patterns-sound-wrong

https://github.com/bangcorrupt/hacktribe/wiki/FAQ#i-cant-import-my-samples
